pkg/codegen/testing/test/testdata/replace-on-change/go/example/cat.go

```diff
@@ -27,9 +27,6 @@ func NewCat(ctx *pulumi.Context,
 		args = &CatArgs{}
 	}
 
-	if args.Name != nil {
-		args.Name = pulumi.ToSecret(args.Name).(pulumi.StringPtrOutput)
-	}
```

Comment:
Does this mean we can delete the `SkipCompileCheck` for Go for this test now?

https://github.com/pulumi/pulumi/blob/a1e18dae4dc00142c086fcae0a90171f278db166/pkg/codegen/testing/test/sdk_driver.go#L158